src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectgrid/view/ProjectGridView.kt

```diff
@@ -44,46 +46,40 @@ class ProjectHomeView : View() {
                 leftAnchor = 0
                 rightAnchor = 0
             }
-            content = flowpane {
-                addClass(AppStyles.appBackground)
-                addClass(ProjectHomeStyles.projectsFlowPane)
-                bindChildren(viewModel.projects) {
+            content =
                     hbox {
-                        projectcard(it) {
-                            addClass(ProjectHomeStyles.projectCard)
-                            titleLabel.addClass(ProjectHomeStyles.projectCardTitle)
-                            languageLabel.addClass(ProjectHomeStyles.projectCardLanguage)
-                            cardButton.apply {
-                                text = messages["loadProject"]
-                                action {
-                                    viewModel.openProject(it)
-                                }
-                            }
-                            deleteButton.apply {
-                                action {
-                                    error(
-                                            messages["deleteProjectPrompt"],
-                                            messages["deleteProjectDetails"],
-                                            ButtonType.YES,
-                                            ButtonType.NO,
-                                            title = messages["deleteProjectPrompt"]
-                                    ) { button: ButtonType ->
-                                        if (button == ButtonType.YES) {
-                                            viewModel.deleteProject(it)
-                                            cardButton.isDisable = true
-                                            isDisable = true
+                        addClass(AppStyles.appBackground)
+                        flowpane {
+                            hgrow = Priority.ALWAYS
+                            addClass(AppStyles.appBackground)
+                            addClass(ProjectGridStyles.projectsFlowPane)
+                            bindChildren(viewModel.projects) {
+                                hbox {
```

Comment:
is there a reason for the hbox here over just the card?

Reply:
for right now the only purpose it serves is to allow you to change the card's base top color depending on whether the card isActive or isComplete. I can remove them but i'd have to get the color to changing to work again another time. the cards would still work